python/pyarrow/tests/interchange/test_conversion.py

```diff
@@ -217,19 +170,28 @@ def test_roundtrip_pandas_string():
 
 
 @pytest.mark.pandas
-def test_roundtrip_pandas_boolean():
-    if Version(pd.__version__) < Version("1.5.0"):
-        pytest.skip("__dataframe__ added to pandas in 1.5.0")
+def test_pandas_roundtrip_categorical():
+    if Version(pd.__version__) < Version("2.0.2"):
```

Comment:
Should the pandas version for skipping tests due to `Column.size() bug in pandas` match between tests `test_pandas_roundtrip_string` (v2.0.1) and `test_pandas_roundtrip_categorical` (v2.0.2)?

Reply:
Good catch! =) There are also bitmasks involved here, so I have to change the skip message 😊

Reply:
I have, unfortunately, added this change in the rebasing process and is therefore not visible in the commit history.
The change can be seen here:
https://github.com/apache/arrow/blob/0e74ace51b3668fca155163e0f887d83ba778ed5/python/pyarrow/tests/interchange/test_conversion.py#L243-L245